app/stories/components/Dropdown.tsx

```diff
@@ -0,0 +1,20 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import DropdownComponent from '@components/Dropdown'
+
+export default {
+  title: 'Components',
+  component: DropdownComponent,
+} as ComponentMeta<typeof DropdownComponent>
+
+const Template: ComponentStory<typeof DropdownComponent> = (args) => (
+  <DropdownComponent {...args} />
+)
+
+export const Dropdown = Template.bind({})
+
+Dropdown.args = {
+  id: 'test',
+  label: 'Select',
+  options: ['1', '2', '3'],
+}
```

Comment:
This file needs to be renamed to `Dropdown.stories.tsx`

Reply:
fixed in e5b4ea043dd3979985e5b8abedc7ae9caeb1b01f